web-console/developer/add/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,28 @@
+# Pipelines
+The Pipelines section allows the user to enable a pipeline template for their application, component or service.
+
+## Default state
+A Pipelines section appears in +Add flows (for Import from Git,  Container Image, and From Dockerfile) when the Pipelines operator is installed.
+* When the Pipelines operator is not installed, this section is not displayed
+* The Pipelines section has a “Dev Preview” badge
+* Under this Pipelines section, there is a checkbox control labeled “Add pipeline”
+* The default state of the Add pipeline checkbox is OFF
+
+![Add pipeline](img/+Add_Flow_Pipelines_default.png)
```

Comment:
@beaumorley for all your mocks below, can you isolate the images to the **Pipelines** piece?

Reason being, the other pieces change frequently - case in point Serverless is now Resource :)

Reply:
Good idea. Updated based on this feedback.